retis-events/src/common.rs

```diff
@@ -2,6 +2,17 @@ use std::fmt;
 
 use crate::{event_section, event_type, *};
 
+#[event_section("md_common")]
+pub struct CommonEventMd {
+    pub retis_version: String,
```

Comment:
Yep. I'm also wondering if this should not be named "global" instead of "common" as it's a section not common to all metadata events but just global information available for everyone. WDYT?

Reply:
+1 to not use "common" again :-)